pkgs/os-specific/linux/conky/default.nix

```
@@ -67,22 +67,25 @@ with lib;
 
 stdenv.mkDerivation rec {
   pname = "conky";
-  version = "1.13.1";
+  version = "1.18.0";
 
   src = fetchFromGitHub {
     owner = "brndnmtthws";
     repo = "conky";
     rev = "v${version}";
-    sha256 = "sha256-3eCRzjfHGFiKuxmRHvnzqAg/+ApUKnHhsumWnio/Qxg=";
+    sha256 = "sha256-a0RGgX325NztDcQwg9+ibxOstU0MSS3eSTaljgt9qPQ=";
   };
 
+  # For some reason -Werror is on by default, causing the project to fail compilation.
+  buildPhase = ''
+    make
+  '';
+  NIX_CFLAGS_COMPILE = "-Wno-error";
```

```suggestion
  env.NIX_CFLAGS_COMPILE = "-Wno-error";
```

it builds fine for me without the `buildPhase` part. also since #217206 i believe we prefer to specify `NIX_CFLAGS_COMPILE` via the `env` attrset.

if you could move this down to the same area we set `NIX_LDFLAGS` in this file that'd be nice too. other than that looks good!